.github/workflows/pinot_tests.yml

```diff
@@ -72,16 +72,20 @@ jobs:
     if: github.repository == 'apache/pinot'
     runs-on: ubuntu-latest
     strategy:
+      # Changed to false in order to improve coverage using unsafe buffers
+      fail-fast: false
```

Comment:
We can change this and other introduced `fail-fast: false` once we are happy with the code